src/Agent.Worker/ManagementScriptStep.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Services.Agent.Util;
+using Microsoft.VisualStudio.Services.Agent.Worker.Handlers;
+
+namespace Microsoft.VisualStudio.Services.Agent.Worker
+{
```

Comment:
add a comment to this class. 
// This is for internal testing and is not publicly supported. This will be removed from the agent at a later time.